modules/library/src/main/scala/zio/elasticsearch/ElasticRequest.scala

```diff
@@ -552,45 +556,59 @@ object ElasticRequest {
       extends ElasticRequest[SearchResult]
       with HasFrom[SearchRequest]
       with HasRouting[SearchRequest]
+      with HasSize[SearchRequest]
       with HasSort[SearchRequest]
-      with HasSize[SearchRequest] {
+      with HasSourceFiltering[SearchRequest] {
     def aggregate(aggregation: ElasticAggregation): SearchAndAggregateRequest
 
+    def excludes(fields: String*): SearchRequest
+
     def highlights(value: Highlights): SearchRequest
 
+    def includes(fields: String*): SearchRequest
+
     def searchAfter(value: Json): SearchRequest
   }
 
   private[elasticsearch] final case class Search(
     index: IndexName,
     query: ElasticQuery[_],
     sortBy: Set[Sort],
+    excluded: Option[List[String]],
```

Comment:
I think a `Chunk` would be more appropriate here, especially since we rely on concatenation in `def excludes`, even though most likely `excludes` will not be used multiple times.